.ci_support/linux_64_.yaml

```
@@ -6,3 +6,5 @@ channel_targets:
 - conda-forge main
 docker_image:
 - quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
+- '3.9'
```

Bump Python version to make azure happy (also new min version for MDA 2.8.0)
```suggestion
- '3.10'
```